report/renderer/report-utils.js

```diff
@@ -187,6 +190,75 @@ class ReportUtils {
     return clone;
   }
 
+  /**
+   * Given an audit's details, identify and return a URL locator function that
+   * can be called later with an `item` to extract the URL of it.
+   * @param {LH.FormattedIcu<LH.Audit.Details.TableColumnHeading[]>} headings
+   * @return {{(item: LH.FormattedIcu<LH.Audit.Details.TableItem>): string|undefined}=}
```

Comment:
I think technically you want
```suggestion
   * @return {((item: LH.FormattedIcu<LH.Audit.Details.TableItem>) => string|undefined)=}
```
(I'm kind of surprised the extra `{}` parses)

Reply:
> I think technically...

Actually @brendankenny, _technically_ this is defining a callable object, so it does make sense that it parses, I just don't believe I've ever seen anyone do that in jsdoc before :) Up to you if you want to change it, though personally I find the arrow syntax and no double `{{}}` alternative slightly easier to parse in this context